### ISSUE TITLE
Execution Block ID uid://A001/X15a0/Xba Sgr_A_st_e_03_7M

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#35]
Sgr_A_st_e_03_7M
[uid://A001/X15a0/Xba](https://almascience.org/aq/?result_view=observation&mous=uid://A001/X15a0/Xba)

* [x] Observations completed?
* [ ] Delivered?
* [ ] Downloaded? (specify where)
  * [ ] hipergator
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_e_03_7M/html/) unpacked
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_e_03_7M/html/) Quality Assessment?
* [ ] Imaging: Continuum
* [ ] Imaging: Lines
  * [ ] HCO+
  * [ ] HNCO
  * [ ] H13CN
  * [ ] H13CO+/SiO
  * [ ] Cont 1
  * [ ] Cont 2

**Comment by keflavich** [comment]
spw24 had some divergence; everything else looks OK? 
![image](https://user-images.githubusercontent.com/143715/162590260-691f4cc4-6ddc-4b78-948c-1d21622336d4.png)

**Comment by d-l-walker** [comment]
Agreed, looks good besides the divergence. **I can work on re-imaging SPW 24**.

Note that this SB has an updated version in the queue. Presumably a velocity shift due to some emission near the edges (see below). Even without this shift, there's still a lot of emission that is well captured in the SPWs with this setup.

![uid___A001_X15a0_Xba s38_0 Sgr_A_star_sci spw20 cube I iter1 image spectrum](https://user-images.githubusercontent.com/13015266/167821715-71df7381-a9b6-406a-a89b-8a7a4b653327.png)


Also looks like another SiO maser in SPW 16:
![uid___A001_X15a0_Xba s38_0 Sgr_A_star_sci spw16 cube I iter1 image pbcor sky](https://user-images.githubusercontent.com/13015266/167820422-59817cd7-8957-4173-aeb4-20b78b36f58f.png)

**Comment by d-l-walker** [comment]
@keflavich do you know what the status of the HPG pipeline run is for this? I was trying to do the reclean of SPW 24, but I can't find the calibrated MSes in the `/calibrated/` directory on globus, only the .ms files in the `/working/ `directory.

**Comment by keflavich** [comment]
there are four mses in the working directory, as you said...
```
/orange/adamginsburg/ACES/rawdata/2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_Xba/calibrated/working/uid___A002_Xf49cca_X25403.ms
/orange/adamginsburg/ACES/rawdata/2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_Xba/calibrated/working/uid___A002_Xf53eeb_X17942.ms
/orange/adamginsburg/ACES/rawdata/2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_Xba/calibrated/working/uid___A002_Xf53eeb_X1834c.ms
/orange/adamginsburg/ACES/rawdata/2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_Xba/calibrated/working/uid___A002_Xf53eeb_X1944e.ms
```
but the pipeline run never finished.

The log file is `/blue/adamginsburg/adamginsburg/ACES/logs/run_pipeline_mpi_25194557.log`.  It ends with
```
2022-04-10 04:23:57 INFO: Executing plotms(vis='uid___A002_Xf49cca_X25403.ms', xaxis='azimuth', yaxis='elevation', spw='16:0~0,18:0~0,20:0~0,22:0~0,24:0~0,26:0~0', antenna='0&&*', avgchannel='9000', avgtime='10', coloraxis='field', customflaggedsymbol=True, flaggedsymbolshape='autoscaling', title='Elevation vs Azimuth for uid___A002_Xf49cca_X25403.ms', plotfile='azel.png', showgui=False, clearplots=True)
fuse: failed to exec fusermount: No such file or directory

Cannot mount AppImage, please check your FUSE setup.
You might still be able to extract the contents of this AppImage
if you run it with the --appimage-extract option.
See https://github.com/AppImage/AppImageKit/wiki/FUSE
for more information
open dir error: No such file or directory
slurmstepd: error: *** JOB 25194557 ON c0705a-s25 CANCELLED AT 2022-04-11T08:48:04 ***
```
which indicates that this was run before I implemented the plotms hack-fix that is required for plotms to run on hipergator.

I have to re-run the pipeline =(

**Comment by d-l-walker** [comment]
Alright, thanks for the info!

I actually did grab those 4 MSes without thinking and ran some scripts, but the output image had fluxes that were way too small, so probably didn't have the calibration applied yet? Cleaning looked great, but values were way out. So I'll wait for the pipeline to finish and grab the output MSes and re-rerun.

**Comment by d-l-walker** [comment]
SPW 24 has now been re-cleaned. I used the cycleniter parameter (=100), which again has succeeded in avoiding divergence. I also tweaked some automasking and threshold parameters. The results look good, though the masking is maybe a little too relaxed, but it's acceptable I think. I'll update the cleaning parameters used soon. 

Here's a view of the maximum intensity to compare with the [one in Adam's comment above](https://github.com/ACES-CMZ/reduction_ACES/issues/35#issuecomment-1094119111):
![Screenshot 2022-05-16 at 18 03 59](https://user-images.githubusercontent.com/13015266/168646082-2543c576-21b6-4c2d-a7ad-7442edfe2bd9.png)

@keflavich I've added all of the tclean output files to the `/upload/` folder on globus in a folder called `/Sgr_A_st_e_03_7M_spw24_reclean/`. The destination folder is `/rawdata/2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_Xba/`.
